'comment' is an invalid keyword argument for 'test.fail_without_changes'

Master and minion version 2019.2.0.
> Any parameters in the state return dictionary can be customized by adding the keywords result, comment, and changes.
```
stop state.apply:
  test.fail_without_changes:
    - order: 1
    - comment: Failing highstate to allow minion restart
    - failhard: true
    - watch:
      - service: salt-minion
```
> [ERROR] Unhandled exception running state.apply
 salt.exceptions.SaltInvocationError: 'comment' is an invalid keyword argument for 'test.fail_without_changes'

While it no longer errors, it appears the comment is ignored and replaced with "Failure!".
```
$ salt-call state.low '{"state":"test", "fun":"fail_without_changes", "__id__": "testing", "name":"testing", "comment":"Hello World!", }' --out=highstate
[ERROR] Failure!
local:
    ----------
    __id__:
        testing
    __run_num__:
        0
    __sls__:
        None
    changes:
        ----------
    comment:
        Failure!
    duration:
        0.587
    name:
        testing
    result:
        False
    start_time:
        17:01:44.621198
```

--- a/salt/states/test.py
+++ b/salt/states/test.py
@@ -77,7 +77,9 @@ def succeed_without_changes(name, **kwargs):  # pylint: disable=unused-argument
     name
         A unique string.
     """
-    ret = {"name": name, "changes": {}, "result": True, "comment": "Success!"}
+    comment = kwargs.get("comment", "Success!")
+
+    ret = {"name": name, "changes": {}, "result": True, "comment": comment}
     return ret
 
 
@@ -90,7 +92,9 @@ def fail_without_changes(name, **kwargs):  # pylint: disable=unused-argument
     name:
         A unique string.
     """
-    ret = {"name": name, "changes": {}, "result": False, "comment": "Failure!"}
+    comment = kwargs.get("comment", "Failure!")
+
+    ret = {"name": name, "changes": {}, "result": False, "comment": comment}
 
     if __opts__["test"]:
         ret["result"] = False
@@ -108,7 +112,9 @@ def succeed_with_changes(name, **kwargs):  # pylint: disable=unused-argument
     name:
         A unique string.
     """
-    ret = {"name": name, "changes": {}, "result": True, "comment": "Success!"}
+    comment = kwargs.get("comment", "Success!")
+
+    ret = {"name": name, "changes": {}, "result": True, "comment": comment}
 
     # Following the docs as written here
     # http://docs.saltstack.com/ref/states/writing.html#return-data
@@ -134,6 +140,8 @@ def fail_with_changes(name, **kwargs):  # pylint: disable=unused-argument
     name:
         A unique string.
     """
+    comment = kwargs.get("comment", "Failure!")
+
     ret = {"name": name, "changes": {}, "result": False, "comment": "Failure!"}
 
     # Following the docs as written here

--- a/tests/unit/states/test_test.py
+++ b/tests/unit/states/test_test.py
@@ -36,6 +36,12 @@ class TestTestCase(TestCase, LoaderModuleMockMixin):
             ret.update({"comment": "Success!"})
             self.assertDictEqual(test.succeed_without_changes("salt"), ret)
 
+        with patch.dict(test.__opts__, {"test": False}):
+            ret.update({"comment": "A success comment!"})
+            self.assertDictEqual(
+                test.succeed_without_changes("salt", comment="A success comment!"), ret
+            )
+
     def test_fail_without_changes(self):
         """
             Test to returns failure.
@@ -44,6 +50,12 @@ class TestTestCase(TestCase, LoaderModuleMockMixin):
         with patch.dict(test.__opts__, {"test": False}):
             ret.update({"comment": "Failure!"})
             self.assertDictEqual(test.fail_without_changes("salt"), ret)
+
+        with patch.dict(test.__opts__, {"test": False}):
+            ret.update({"comment": "A failure comment!"})
+            self.assertDictEqual(
+                test.fail_without_changes("salt", comment="A failure comment!"), ret
+            )
 
     def test_succeed_with_changes(self):
         """
@@ -65,6 +77,23 @@ class TestTestCase(TestCase, LoaderModuleMockMixin):
             )
             self.assertDictEqual(test.succeed_with_changes("salt"), ret)
 
+        with patch.dict(test.__opts__, {"test": False}):
+            ret.update(
+                {
+                    "changes": {
+                        "testing": {
+                            "new": "Something pretended" " to change",
+                            "old": "Unchanged",
+                        }
+                    },
+                    "comment": "A success comment!",
+                    "result": True,
+                }
+            )
+            self.assertDictEqual(
+                test.succeed_with_changes("salt", comment="A success comment!"), ret
+            )
+
     def test_fail_with_changes(self):
         """
             Test to returns failure and changes is not empty.
@@ -84,6 +113,23 @@ class TestTestCase(TestCase, LoaderModuleMockMixin):
                 }
             )
             self.assertDictEqual(test.succeed_with_changes("salt"), ret)
+
+        with patch.dict(test.__opts__, {"test": False}):
+            ret.update(
+                {
+                    "changes": {
+                        "testing": {
+                            "new": "Something pretended" " to change",
+                            "old": "Unchanged",
+                        }
+                    },
+                    "comment": "A failure comment!",
+                    "result": True,
+                }
+            )
+            self.assertDictEqual(
+                test.succeed_with_changes("salt", comment="A failure comment!"), ret
+            )
 
     def test_configurable_test_state(self):
         """

